lib/rules/no-unused-vars.js

```diff
@@ -18,26 +18,49 @@ module.exports = {
       category: 'essential',
       url: 'https://eslint.vuejs.org/rules/no-unused-vars.html'
     },
-    fixable: null,
-    schema: []
+    fixable: 'code',
+    schema: [
+      {
+        'type': 'object',
+        'properties': {
+          'varIgnorePattern': {
+            'type': 'string'
+          }
+        },
+        'additionalProperties': false
+      }
+    ]
   },
 
   create (context) {
+    const option = context.options[0] || { varIgnorePattern: '^_' }
```

Comment:
If user don't set option, could you change the default behavior to not ignore everything?